packages/subgraph/schema.graphql

```diff
@@ -540,9 +563,15 @@ type RoleRevokedEvent implements Event @entity {
 type CFAv1LiquidationPeriodChangedEvent implements Event @entity {
     id: ID!
     transactionHash: Bytes!
+    gasPrice: BigInt!
     timestamp: BigInt!
     name: String!
 
+    """
+    The address of the governance contract the event was emitted from.
+    """
+    governanceAddress: Bytes!
```

Comment:
I didn't see this mapped in: `packages/sdk-core/src/mapGetAllEventsQueryEvents.ts`